client/src/pages/trainer/courses/[id]/index.tsx

```diff
@@ -0,0 +1,43 @@
+import ContentSection from '@/src/sections/courses/ContentSection';
+import Breadcrumbs from '@/src/shared/components/Breadcrumbs';
+import Tabs from '@/src/shared/components/Tabs';
+import Tab from '@/src/shared/components/Tabs/Tab';
+import Container from '@/src/shared/layouts/Container';
+import { Fragment } from 'react';
+
+const CourseContent: React.FC = () => {
+  const paths = [
+    {
+      text: 'Course',
+      url: '/'
```

Comment:
```suggestion
      url: '/trainer/courses'
```
since its decided now, we can redirect user to courses list from here